Source/WebCore/rendering/GridTrackSizingAlgorithm.cpp

```diff
@@ -1389,8 +1389,38 @@ void GridTrackSizingAlgorithm::accumulateIntrinsicSizesForTrack(GridTrack& track
             RenderGrid* inner = downcast<RenderGrid>(gridItem);
             GridSpan span = m_renderGrid->gridSpanForChild(*gridItem, m_direction);
 
+            auto extraMarginFromSubgridAncestorGutters = [&]() -> std::optional<LayoutUnit> {
+                if (span.startLine() != trackIndex && span.endLine() - 1 != trackIndex)
```

Comment:
Do we not need to add the gutter padding for tracks that are in the middle of the sub gridded span?

Reply:
Based off my current reading of the spec I don't *think* so:

> Meanwhile, half the size of the difference between the [subgrid](https://drafts.csswg.org/css-grid-2/#subgrid)’s [gutters](https://drafts.csswg.org/css-align-3/#gutter) ([row-gap](https://drafts.csswg.org/css-align-3/#propdef-row-gap)/[column-gap](https://drafts.csswg.org/css-align-3/#propdef-column-gap)) and its [parent grid](https://drafts.csswg.org/css-grid-2/#parent-grid)’s gutters is applied as an extra layer of (potentially negative) margin to the items not at those edges.

Since we're adding extra margins in particular to the subgrids, I think we only want to affect the tracks where the edges are